raiden/tests/integration/rpc/RpcTest.sol

```diff
@@ -1,19 +1,27 @@
 pragma solidity ^0.4.0;
```

Comment:
Do we need to upgrade the solc version in the pragma for the pure public keywords?

Reply:
Good catch, should be 0.4.16. Fixed.